sandbox/src/mocks/changeShortlistAndSendForTriage/responses/ReferralRequest.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
I think the response to the `ChangeShortlistAndSendForTriage` endpoint should be returning a `Parameters` resource rather than a `ReferralRequest`. This way we can return both the updated `ReferralRequest` and the newly created `Appointment`.